util/changeref.py

```diff
@@ -20,7 +22,12 @@
 
 if new_ref_type not in ["ref", "any", "doc"]:
     print("Third arg not a valid ref type string.")
-    sys.exit()    
+    sys.exit()  
+
+redirect = "  %s: %s\n" %(old_path, new_path)
```

Comment:
We might call this a naive implementation.

It depends on the writer's ability (and willingness) to construct the entire redirect string, which might be from a page to a section, or from a section to a page, and include `#reference-targets` that have a different name than the section labels in the code. 

We just need to think about this a little more.

Reply:
I felt the same here but the issue here is how do we construct the redirect string internally and what inputs do we need from the writer. We cannot construct the string without the section labels, doc info. So, we would require all of this as input or the complete redirect string.